src/Raven.Server/Documents/PeriodicBackup/BackupTask.cs

```diff
@@ -624,18 +627,24 @@ private class InternalBackupResult
                 }
             }
 
+            if (_backupToLocalFolder)
+            {
```

Comment:
Won't this (and other) run on incremental backups?

Reply:
Done